shared/store/enhancer.platform.desktop.js

```diff
@@ -3,7 +3,7 @@ import {batchedSubscribe} from 'redux-batched-subscribe'
 import {compose, applyMiddleware} from 'redux'
 import throttle from 'lodash/throttle'
 
-const throttleNotify = throttle(notify => notify(), 200)
+const throttleNotify = throttle(notify => notify(), 100)
```

Comment:
reduce this, we should really try and move towards a world where this is removed